spyder/config/main.py

```diff
@@ -435,6 +435,26 @@
               'editor/split vertically': "Ctrl+{",
               'editor/split horizontally': "Ctrl+_",
               'editor/close split panel': "Alt+Shift+W",
+              "editor/bookmark current position to slot 0": "Ctrl+Shift+0",

```

Comment:
Can you greatly shorten this, e.g. just ``Save bookmark N``

Reply:
I'm not exactly sure what you mean here. If I only use this line
`"editor/Save bookmark N": "Ctrl+Shift+N"`
this can't be linked to all shortcuts right?
Or do you mean to just make the string itself shorter, from
`bookmark current position to slot 0` to `Save bookmark N`?

Reply:
I meant to make the string itself shorter, as it is far too long as is.

Reply:
Agreed, I changed them to your suggestions.